src/common/libutil/cleanup.h

```diff
@@ -8,6 +8,9 @@
  * SPDX-License-Identifier: LGPL-3.0
 \************************************************************/
 
+#ifndef HAVE_LIBUTIL_CLEANUP_H
+#define HAVE_LIBUTIL_CLENAUP_H 1
```

Comment:
typo: "CLENAUP"

Reply:
Ah, thanks. Force pushed a fix. I was kind of hoping lgtm scan would catch that sort of thing, but oh well.